scalableminds/webknossos
# Allow deactivated users to switch organization

Fixed a bug where a user with multiple organizations could not log in anymore after one of their organization accounts got deactivated.

### Steps to test (I did this already):
- Create a second organization
- Invite a user from orga A into orga B
- accept the invite, switching to orga B
- then as admin of orga B, deactivate that user in your orga again
- after logout + login, the user should be back in orga A

### Issues:
- fixes #6100

------
